editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -1727,6 +1727,13 @@ void Node3DEditorViewport::_sinput(const Ref<InputEvent> &p_event) {
 					_edit.mode = TRANSFORM_TRANSLATE;
 				}
 
+				// enable region-select if nothing has been selected yet or multi-select (shift key) is active
+				if (movement_threshold_passed && (!get_selected_count() || clicked_wants_append)) {
+					cursor.region_select = true;
+					cursor.region_begin = _edit.original_mouse_pos;
+					cursor.region_end = m->get_position();
```

Comment:
Another nitpick: `cursor.region_end = m->get_position();` is already done in line 1738 below, so it doesn't need to be added here.

Reply:
Good point! I removed it.